packages/scandipwa/src/component/MyAccountDashboard/MyAccountDashboard.container.js

```diff
@@ -36,7 +36,6 @@ export class MyAccountDashboardContainer extends PureComponent {
 
     containerProps() {
         const { customer } = this.props;
-
```

Comment:
Please put the empty line between the `return` statement and the rest of the code.